lib/digest/crc15.rb

```diff
@@ -1,5 +1,15 @@
 require 'digest/crc'
 
+class Digest::CRC15 < Digest::CRC; end
```

Comment:
Without this definitions we have 
```
irb(main):001:0> require 'digest/crc32'
/home/ojab/.rbenv/versions/3.0.1/lib/ruby/3.0.0/digest.rb:17: warning: /home/ojab/.rbenv/versions/3.0.1/lib/ruby/3.0.0/digest.rb:17: warning: loading in progress, circular require considered harmful - /home/ojab/src/digest-crc/lib/digest/crc32.rb
	from /home/ojab/.rbenv/versions/3.0.1/bin/bundle:23:in  `<main>'
	from /home/ojab/.rbenv/versions/3.0.1/bin/bundle:23:in  `load'
```
on binary ext require. Have no idea how this stuff works.